controllers/platformnetwork_controller.go

```diff
@@ -317,6 +414,18 @@ func (r *PlatformNetworkReconciler) ReconcileAddressPool(client *gophercloud.Ser
 	} else {
 		if pool == nil {
 			pool, err = r.ReconcileNewAddressPool(client, instance)
+		} else if instance.Spec.Type == "oam" {
```

Comment:
Personally, I prefer to use Constant for "oam" but not mandatory

Reply:
We'll handle this in a follow-up commit when we create and manage the other networks (admin, mgmt). Is that okay?